### ISSUE TITLE
Ansible and Ansible Galaxy unattended upgrades

### ISSUE THREAD

**Issue by gregharvey** [codeenigma/ansible-provision#1038]
Ansible periodically updates namespaces as modules get passed between projects or renamed (mostly it's community managed AWS modules moving to the official amazon namespace). When these changes happen it triggers errors in `ansible-lint`, however fixing the errors requires latest collections to be installed on all controller and deploy servers.

On our test controller I did this:

```
sudo su controller
/usr/bin/python -m pip install --upgrade pip
/usr/bin/python -m pip install --upgrade ansible
ansible-galaxy collection install amazon.aws --force
ansible-galaxy collection install community.aws --force
```

We have a list of collections in the meta files:
* https://github.com/codeenigma/ce-provision/blob/1.x/roles/ce_provision/meta/requirements.yml
* https://github.com/codeenigma/ce-provision/blob/1.x/roles/ce_deploy/meta/requirements.yml

There's also code for handling user-defined "extras":
* https://github.com/codeenigma/ce-provision/blob/1.x/roles/ce_provision/tasks/main.yml#L151
* https://github.com/codeenigma/ce-provision/blob/1.x/roles/ce_deploy/tasks/main.yml#L153

So we could create cron jobs by iterating over those entries, should be reasonably straightforward to achieve.

This is less of an issue with containerised deployers/controllers as they can simply be repacked on demand, but still, it would be good to have these things getting routinely upgraded.

**Comment by gregharvey** [comment]
WIP:
* https://github.com/codeenigma/ce-provision/pull/1037/files

**Comment by gregharvey** [comment]
Fixed in #1037.
